LightBlue/Hosts/WorkerHost.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Configuration;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using LightBlue.Setup;
+using Microsoft.WindowsAzure.ServiceRuntime;
+using Serilog;
+
+namespace LightBlue.Hosts
+{
+    public class WorkerHost
+    {
+        private readonly Settings _settings;
+        private RoleEntryPoint _role;
+        private Task _task;
+
+        public WorkerHost(Settings settings)
+        {
+            _settings = settings;
+        }
+
+        public void Start()
+        {
+            try
+            {
+                Log.Logger = new LoggerConfiguration()
+                    .WriteTo.LiterateConsole()
+                    .WriteTo.RollingFile(@"c:\LightBlueLogs\" + _settings.ServiceTitle + "-{Date}.txt")
+                    .CreateLogger();
+
+                Log.Information("Worker host service current directory {CurrentDirectory}", Directory.GetCurrentDirectory());
+
+                var hostDirectory = LightBlueConfiguration.SetAsWindowsHost(_settings.ServiceTitle,
```

Comment:
SetAsWindowsHost is a new method for running as a windows service.  It sets the data directory and creates a LightBlueAppDomainContext